presto-docs/src/main/sphinx/connector/system.rst

```diff
@@ -34,6 +34,11 @@ System Connector Tables
 
 The catalogs table contains the list of available catalogs.
 
+``metadata.table_properties``
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
This needs to be the same length as the text

I think the Python IDEA plugin will warn